onedal/basic_statistics/tests/test_basic_statistics.py

```diff
@@ -14,155 +14,222 @@
 # limitations under the License.
 # ==============================================================================
 
+import numpy as np
+import pytest
+from numpy.testing import assert_allclose
```

Comment:
Maybe I am missing something but it looks like CSR data testing was deleted from this file in this PR.
Need to test scipy.csr_array inputs as well, not only Numpy.

Reply:
looks like rebase issue, must be fixed now. thanks for pointing out.